tests/test_openclhpp.cpp

```diff
@@ -153,13 +153,13 @@ static cl_int clGetPlatformInfo_version(
 {
     size_t bytes = strlen(version) + 1;
 
-    TEST_ASSERT_NOT_NULL(id);
+    TEST_ASSERT_NOT_nullptr(id);
```

Comment:
I think your search-and-replace was a little too aggressive :smiley:.

These should all remain `TEST_ASSERT_NOT_NULL`.

Reply:
Thanks for this!
Just fixed